### PR TITLE
Fixed Heartbeat node's translation

### LINKED ISSUE #7580: Script Canvas: The HeartBeat node is missing its header
**Describe the bug**
The HeartBeat node from Timing category does not have its header in the Script Canvas.

**Steps to reproduce**
1. Launch the Script Canvas tool.
2. Add the HeartBeat node to the graph.

**Expected behavior**
All Script Canvas nodes have their headers.

**Actual behavior**
The HeartBeat node is missing its header.

**Screenshot**
HeartBeat_Node

**Found in Branch**
Development (257f3cb)

**Desktop/Device:**
 - Device: PC
 - OS: Linux
 - Version: Ubuntu 20.04.3 LTS
 - CPU: AMD Ryzen 5 3600
 - GPU: NVIDIA GeForce RTX 2060 SUPER
 - Memory: 16GB

### PR DESCRIPTION
**Pull request by lsemp3d**
Closes #7580 

Signed-off-by: lsemp3d <58790905+lsemp3d@users.noreply.github.com>

### CODE REVIEW

File: Gems/ScriptCanvas/Assets/TranslationAssets/Nodes/Timing_HeartBeat.names [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
     "entries": [
         {
-            "base": "{BA107060-249D-4818-9CEC-7573718273FC}",
+            "base": "{E73DB180-A325-763B-A1FE-517B548AF66E}",
```
Comment (onecent1101) [inline Gems/ScriptCanvas/Assets/TranslationAssets/Nodes/Timing_HeartBeat.names:4]: So the missing header is caused by the wrong uuid? assume the rest change is not related to GHI?

**Review by onecent1101 — APPROVED** [review]

**Review by carlitosan — APPROVED** [review]

File: Gems/ScriptCanvas/Assets/TranslationAssets/Nodes/Timing_HeartBeat.names [hunk 2]
```diff
@@ -16,22 +16,35 @@
...
+                },
                 {
                     "base": "Input_Stop_1",
                     "details": {
                         "name": "Stop"
                     }
                 },
                 {
-                    "base": "Output_Pulse_0",
+                    "base": "Output_On Stop_1",
```
[the remark this answers is not in the document]
Comment (lsemp3d) [reply Gems/ScriptCanvas/Assets/TranslationAssets/Nodes/Timing_HeartBeat.names:38]: good observation, in the end it's a string compare and should be ok, that field is auto-generated

File: Gems/ScriptCanvas/Assets/TranslationAssets/Nodes/Timing_HeartBeat.names (hunk 1 above)
Comment (lsemp3d) [reply Gems/ScriptCanvas/Assets/TranslationAssets/Nodes/Timing_HeartBeat.names:4]: I regenerated the file, I hadn't regenerated it after making changes to the generation algorithm, so it picked up a few improvements as well
